### PR TITLE
[14][MIG] sale_shipping_info_helper

### PR DESCRIPTION
**Pull request by bguillot**
Migration of sale shipping info helper

### CODE REVIEW

File: sale_shipping_info_helper/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @api.depends("amount_total", "amount_untaxed")
+    def _compute_shipping(self):
+        for record in self:
+            record.shipping_amount_untaxed = (
+                record.shipping_amount_total
+            ) = record.shipping_amount_tax = 0
+            for line in record.order_line:
+                if not line.is_delivery:
+                    continue
+                record.update(
```
Comment (atchuthan) [inline sale_shipping_info_helper/models/sale_order.py]: If we have partial DO and shipped later, then we might have multiple shipping line, so we need to append the values of each shipping line right?

**Review by atchuthan — CHANGES_REQUESTED** [review]

**Review by dsolanki-initos — APPROVED** [review]
Code review no major change.

File: sale_shipping_info_helper/models/sale_order.py (hunk 1 above)
Comment (bguillot) [reply sale_shipping_info_helper/models/sale_order.py]: Indeed you can have several shipping lines.

Thanks

**Review by kevinkhao — APPROVED** [review]
Code LGTM

**Comment by dsolanki-initos** [comment]
It would be great if somone can merge this PR.

**Review by atchuthan — APPROVED** [review]

**Pull request merged**